src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -26,6 +26,7 @@ internal class AzureAppConfigurationProvider : ConfigurationProvider, IConfigura
     {
         private bool _optional;
         private bool _isInitialLoadComplete = false;
+        private bool _featureManagementVersionsTelemetrySet;
```

Comment:
How about `_isFeatureManagementVersionInspected`?

Reply:
I think that sounds fine, but @jimmyca15 would you still prefer `_isTelemetrySet` once I pull the logic out to an `EnsureTelemtry` method?

Reply:
We should use "tracing" instead of "telemetry". While I was thinking about suggesting that, I figured we could avoid it altogether :)

Reply:
Name update sounds good.